src/xdocs/config_metrics.xml

```diff
@@ -111,6 +127,22 @@
   &lt;property name=&quot;max&quot; value=&quot;7&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>Code Example:</p>
+          <div class="wrapper">
+            <pre>
+              public class Test
+              {
+                public static void main(String ... args)
+                {
+                  boolean a = true;
+                  boolean b = false;
+
+                  boolean c = (a &amp; b) | (b &amp; a) ^ (a &amp; b) | (b &amp; a); //OK
+                  boolean d = (a|b)^(a|b)^(a || b)|(b &amp; a)^ b; //violation,complexity is 8
```

Comment:
Please space out this line so it is easier to read. Everything is so compact. Line is allowed to span multiple if needed.

Reply:
done